### PR TITLE
media-plugins/alsaequal: Fix https://bugs.gentoo.org/499298 reported by Tiago Resende

### PR DESCRIPTION
**Pull request by bassdr**
Bug: https://bugs.gentoo.org/499298

Package-Manager: Portage-2.3.13, Repoman-2.3.3

### CODE REVIEW

**Comment by flappyports** [comment]
@bassdr Do you still intend to work on this PR?

**Comment by bassdr** [comment]
Yes, why not? I did not have any feedback though... I am using alsaequal on one of my box and it works using my fork...

File: media-plugins/alsaequal/alsaequal-0.7_alpha.ebuild [hunk 1]
```diff
@@ -1,16 +1,16 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
```
Comment (flappyports) [inline media-plugins/alsaequal/alsaequal-0.7_alpha.ebuild]: Update the copyright

File: media-plugins/alsaequal/alsaequal-0.7_alpha.ebuild [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=5
 inherit eutils multilib toolchain-funcs multilib-minimal
 
-DESCRIPTION="a real-time adjustable equalizer plugin for ALSA"
-HOMEPAGE="http://www.thedigitalmachine.net/alsaequal.html"
-SRC_URI="http://www.thedigitalmachine.net/tools/${P}.tar.bz2"
+DESCRIPTION="A real-time adjustable equalizer plugin for ALSA"
+HOMEPAGE="https://github.com/bassdr/alsaequal"
```
Comment (flappyports) [inline media-plugins/alsaequal/alsaequal-0.7_alpha.ebuild]: Why not use ${PN} here as well?

**Comment by flappyports** [comment]
No activity in 45+ days.

**Pull request closed without merging**